pkg/metrics/controller_metrics.go

```diff
@@ -111,6 +113,19 @@ var (
 			Aggregation: view.Count(),
 			TagKeys:     []tag.Key{keyType, keyFleetName},
 		},
+		&view.View{
```

Comment:
not sure if this is needed, the distribution bucket #0 will have the same value.

Reply:
Can I get the count of nodes currently in the cluster using the distribution ? I can get the average using _sum and _count but not sure about instant value.

Reply:
You can’t since the sum is ever increasing. A gauge is required.

Reply:
Ok so I’ll keep the view.